<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditServicesDialog</class>
 <widget class="QDialog" name="AddEditServicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    height: 500px;
    width: 400px;
    background-color: #b59683;
  }
  
  #frame {
    border: 3px solid #ebebd8;
    border-radius: 15%;
    background-color: #ebebd8;
  }
  
  QLabel {
      color: #b59683;
      font-weight: bold;
  }
  
  QLabel[objectName*=&quot;title&quot;] {
    padding: 5px;
    
    font-size: 42px;
    font-weight: bold;
  
    color: #eeeee4;
    border-radius: 15%;
    background-color: #b59683;
    
    margin-bottom: 10px;
  }
  
  QLabel[objectName*=&quot;image&quot;] {
        min-height: 150px;
        min-width: 150px;
    
        border: solid;
      border-width: 3px;
      border-radius:15%;
      border-color: #b59683;
  
      background-color: #eeeee4;
  }
  
  QPushButton {
      width: 90;
      height: 20;
      padding: 5px;
  
      font: bold;
  
      color: #b59683;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QPushButton::pressed {
        color: #eeeee4;
        background-color: #b59683;
  }
  
  QPushButton[objectName*=&quot;save&quot;], QPushButton[objectName*=&quot;upload&quot;]
  {
    width: 90;
    height: 20;
    padding: 5px;
  
    font: bold;
  
    color: #eeeee4;
    border: solid;
    border-width: 5px;
    border-radius:20px;
    border-color: #284256;
    background-color: #284256;
  }
  
  QPushButton[objectName*=&quot;save&quot;]::pressed, QPushButton[objectName*=&quot;upload&quot;]::pressed {
    color: #284256;
    background-color: #eeeee4;
  }
  
  QLineEdit {
      width: 200;
      height: 20;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QLineEdit::focus {
      border-color: #284256;
  }
  
  QSpinBox, QDoubleSpinBox {
      width: 200px; /* Добавляем единицу измерения (px) к значению ширины */
      height: 20px;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: 5px solid #b59683; /* Устанавливаем толщину границы */
      border-radius: 20px;
      background-color: #eeeee4;
  }
  
  QSpinBox::focus, QDoubleSpinBox::focus {
    border-color: #284256;
  }
  
  QDoubleSpinBox::up-button, QDoubleSpinBox::down-button, QSpinBox::up-button, QSpinBox::down-button {
    width: 0;
    height: 0;
  }
  
  QPlainTextEdit {
      width: 200;
      height: 100;
      padding: 5px;
  
      font: bold;
  
      color: #284256;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QPlainTextEdit::focus {
      border-color: #284256;
  }
  
  QComboBox {
    width: 150;
      height: 20;
      padding: 5px;
  
      font: bold;
  
      color: #b59683;
      border: solid;
      border-width: 5px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }
  
  QComboBox:hover {
  
  }
  
  QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border-left: 5px solid #b59683;
  }
  
  QComboBox::down-arrow {
      
  }
  
  QComboBox QListView {
      padding: 5;
  }
  
  QComboBox::selected {
      color: #ebebd8;
      background-color: #284256;
  }
  
  QAbstractItemView {
    border: 3px solid #b59683;
    border-radius: 15%;
    background-color: #eeeee4;
  }
  
  QAbstractItemView::selected {
      color: #eeeee4;
      background-color: #284256;
  }
  
  #frame_cost {
    padding: 0px;
    border: solid;
      border-width: 3px;
      border-radius:20px;
      border-color: #b59683;
      background-color: #eeeee4;
  }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>Додавання</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">#frame_2 {
color: #ebebd8;
  border-radius: 15%;
  background-color: #b59683;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_image">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Зображення</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_upload">
           <property name="text">
            <string>Завантажити</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="comboBox_type">
          <property name="placeholderText">
           <string>Тип послуги</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_title">
          <property name="placeholderText">
           <string>Назва</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_description">
        <property name="placeholderText">
         <string>Опис</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Вартість:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_cost">
          <property name="maximum">
           <double>99999999.989999994635582</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="text">
           <string>Скасувати</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Зберегти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
